packages/garbo/src/resources/fax.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  canFaxbot,
+  cliExecute,
+  faxbot,
+  getClanLounge,
+  Monster,
+  wait,
+} from "kolmafia";
+import { $item, get, have, property } from "libram";
+
+function checkFax(monster: Monster): boolean {
+  if (!have($item`photocopied monster`)) cliExecute("fax receive");
+  if (!have($item`photocopied monster`)) return false;
+  if (property.get("photocopyMonster") === monster) {
```

Comment:
Is it worth adding an `examine(photocopied monster)` here?

Reply:
Worth noting: Even if mafia always does it as part of the command, it may not examine the monster on login and so if you start your session with a fax it may still break